plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -476,14 +486,14 @@ def _onSceneChanged(self, source: SceneNode) -> None:
         else:
             # we got a single scenenode
             if not source.callDecoration("isGroup"):
-                if source.getMeshData() is None:
-                    return
-                if source.getMeshData().getVertices() is None:
+                meshData = source.getMeshData();
```

Comment:
Use correct naming for variables (`mesh_data`) and remove `;`

Reply:
Fixed in ac0192f